app/controllers/users/lists_controller.rb

```diff
@@ -2,25 +2,31 @@ class Users::ListsController < ApplicationController
   before_action :ensure_current_user, except: [:index, :show]
   before_action :set_user
   before_action :set_list, except: :index
+  before_action :ensure_editable, only: [:edit, :update, :destroy]
+  before_action :ensure_visible, only: :show
 
   def index
-    @lists = @user.lists
+    @lists = @user.authored_lists.uniq
     render 'lists/index'
   end
 
-  def edit
-    render 'lists/edit'
-  end
-
   def show
     @references = @list.references.joins(:paper).order(params_sort_order)
     render 'lists/show'
   end
 
+  def edit
+    render 'lists/edit'
```

Comment:
should we do any rejiggering of our views folders to make these fall in line with rails defaults, ie put this view in users/lists/edit?  I'm not sure that will even work.

I've also been thinking about how we should move the partials around, I'm always a little confused as to when to put a partial into papers or references folders.